test/CheckoutSdk.Tests/Payments/GetTests.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Checkout.Common;
+using Checkout.Payments;
+using Shouldly;
+using Xunit;
+
+namespace Checkout.Tests.Payments
+{
+    public class GetTests : ApiTestFixture
+    {
+        [Fact]
+        public async Task CanGetCardPayment()
+        {
+            PaymentProcessed payment = await MakeCardPaymentAsync();
+
+            GetPaymentResponse verifiedPayment = await Api.Payments.GetAsync(payment.Id);
+
+            verifiedPayment.ShouldNotBeNull();
+            verifiedPayment.Id.ShouldBe(payment.Id);
+            verifiedPayment.Source.AsCard().Id.ShouldBe(payment.Source.AsCard().Id);
+        }
+
+        public static IEnumerable<object[]> AlternativePaymentMethodsTestData =>
+            new List<object[]>
+            {
+                new object[] { new AlternativePaymentSource("giropay") { { "bic", "TESTDETT421" }, { "purpose", "CKO giropay test" } }, Currency.EUR },
+                new object[] { new AlternativePaymentSource("ideal") { { "issuer_id", "INGBNL2A" } }, Currency.EUR }
+            };
+
+        [Theory]
+        [MemberData(nameof(AlternativePaymentMethodsTestData))]
+        public async Task CanGetAlternativePayment(IRequestSource alternativePaymentMethodRequestSource, string currency)
```

Comment:
I would prefer for us to have specific tests for each payment method since it's far more explicit should a test fail